polkadot/node/subsystem-test-helpers/src/lib.rs

```diff
@@ -435,42 +435,6 @@ impl Future for Yield {
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use futures::executor::block_on;
-	use polkadot_node_subsystem::messages::CollatorProtocolMessage;
-	use polkadot_overseer::{dummy::dummy_overseer_builder, Handle, HeadSupportsParachains};
-	use polkadot_primitives::Hash;
-	use sp_core::traits::SpawnNamed;
-
-	struct AlwaysSupportsParachains;
-
-	#[async_trait::async_trait]
-	impl HeadSupportsParachains for AlwaysSupportsParachains {
-		async fn head_supports_parachains(&self, _head: &Hash) -> bool {
-			true
-		}
-	}
-
-	#[test]
-	fn forward_subsystem_works() {
```

Comment:
I had to remove this test to break (dev-) cycle dependency. Also, in general, I am not convinced we need tests for tests.